lib/diplomat/datacenter.rb

```diff
@@ -4,19 +4,19 @@ class Datacenter < Diplomat::RestClient
     @access_methods = [:get]
 
     # Get an array of all avaliable datacenters accessible by the local consul agent
-    # @param meta [Hash] output structure containing header information about the request (index)
+    # @param options [Hash] options parameter hash
     # @return [OpenStruct] all datacenters avaliable to this consul agent
-    def get(meta = nil)
-      url = ['/v1/catalog/datacenters']
+    # rubocop:disable AbcSize, CyclomaticComplexity
+    def get(options = nil)
```

Comment:
This is a breakage of API for those using meta to get information about request.

Reply:
Yes indeed. I've fixed this issue.